### ISSUE TITLE
[a11y-defect-3]: Accessible labels for `va-pagination` Previous and Next links must be internationalized

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96438]
### Point of contact

Trevor Pierce

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

While making a PR to refactor `<va-pagination>` links and ellipses for accessibility, I noticed the Next and Previous links were not being internationalized in the [Spanish demo](https://design.va.gov/components/pagination#internationalization). This issue is being filed to ask that new definitions be added and the `aria-label` attributes use those internationalized strings instead of hard-coded values. Screenshot attached below.

---
![Image](https://github.com/user-attachments/assets/81361ce3-2d1c-4a57-8dd4-165e003272d3)

### Reproduction steps

1. View https://design.va.gov/components/pagination#internationalization with a screen reader enabled
2. Click the `Espanol` button to switch to Spanish. You'll only hear the visible text, then "Link".
3. We want it to read out "Next page, Link" or "Previous page, Link" in Spanish. It's possible only the text string will be updated. Link may be out of scope because that's browser specific.

### Proposed solution or next steps

Once my PR https://github.com/department-of-veterans-affairs/component-library/pull/1388 is merged or rejected, I'd ask the Design System team to consider making this change. Ideally the `aria-label` will stay on the A tag as I've suggested, for maximum usability. Adding a properly internationalized string to the accessible labels is another win for usability.

### References, articles, or WCAG support

1. [aria-label Does Not Translate](https://adrianroselli.com/2019/11/aria-label-does-not-translate.html#Update06)
2. Understanding [SC 2.4.4](https://w3.org/TR/WCAG22#link-purpose-in-context): Link Purpose (In Context) (Level A)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [x] Something else
